Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -686,7 +686,7 @@ - (void)updateUser:(NSDictionary *)dataFields mergeNestedObjects:(BOOL)mergeNest
 // documented in IterableAPI.h
 - (void)updateEmail:(NSString *)newEmail onSuccess:(OnSuccessHandler)onSuccess onFailure:(OnFailureHandler)onFailure
 {
-    if (newEmail) {
+    if (_email && newEmail) {
```

Comment:
when should we use `_email` vs `self.email`?

Reply:
Doesn't really matter unless getters/setters are overridden; my personal preference is to use `self.` (the method call) most of the time, but the SDK already uses fields.

Reply:
Xcode linter also asks to explicitly do `this->_email` or `this.email` when using instance properties/fields in a block which causes it to retain `this`.

Reply:
i see. i was asking because 2 lines down i see `self.email`...
we should probably stick to the same style throughout the code base (i don't care which one you pick, just a thought)